ReactiveProducers/ReactiveKafkaDumpProducer/src/main/resources/application.conf

```diff
@@ -0,0 +1,13 @@
+
+
+dataDir=${HOME}/Dev/data/ld4pData
+dataDir=${?ld4pData}
```

Comment:
The convention for this ENV variable is now `LD4P_DATA`.  Please update to match.

Reply:
@jgreben Looks like that might be fixed if you just rebase.